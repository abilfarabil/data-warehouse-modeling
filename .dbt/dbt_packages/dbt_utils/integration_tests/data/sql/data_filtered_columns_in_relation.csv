version https://git-lfs.github.com/spec/v1
oid sha256:0ef5f4490e603dbbc27942e2edeb7c8243dba159e1c4608a120688c84cde7d93
size 44
