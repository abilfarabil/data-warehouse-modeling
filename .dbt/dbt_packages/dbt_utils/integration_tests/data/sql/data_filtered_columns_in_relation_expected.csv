version https://git-lfs.github.com/spec/v1
oid sha256:b5dd68b593c02823f6cfbff2f6a11b57307d28e215667e427bd733a86520e38a
size 20
